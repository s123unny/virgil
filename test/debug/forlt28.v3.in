b ../core/forlt28.v3 5
run
bt
info l
c
q
