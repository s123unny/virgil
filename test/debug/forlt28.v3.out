breakpoint|../core/forlt28.v3 5|0
stop|stopOnBreakpoint
bt|main|../core/forlt28.v3|5
variable|0|a|0|int|false
variable|1|m|#1:Array<int -> void>|Array<int -> void>|true
stop|end
result|0
